build_carddb_lorcana.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+"""
+This script downloads the latest Disney Lorcana card data from https://lorcanajson.org/ and processes
+it to turn the highly-structured data there into a flat list of card names to descriptions
+formatted to send down the chat.
+"""
+
+import common
+common.FRAMEWORK_ONLY = True
+
+import json
+import sys
+import zipfile
+
+import psycopg2
+
+from common.card import CARD_GAME_LORCANA
+from common.config import config
+import build_carddb_mtg
+
+def main():
+	forceRun = '-f' in sys.argv
+
+	was_downloaded = build_carddb_mtg.do_download_file("https://lorcanajson.org/files/current/en/allCards.json.zip", "LorcanaCards.json.zip")
```

Comment:
Instead of importing it from `build_carddb_mtg.py` `do_download_file()` should be moved to `common/http.py` and imported from both the scripts.

Reply:
I considered that, but I wanted to keep my changes as small as possible. But you're right, and I moved it now